wfe/wfe.go

```diff
@@ -453,6 +454,11 @@ func (wfe *WebFrontEndImpl) verifyPOST(ctx context.Context, logEvent *requestEve
 		logEvent.Contacts = reg.Contact
 	}
 
+	if wfe.AllowAccountDeactivation && reg.Status != core.StatusValid {
+		fmt.Println(reg)
+		return nil, nil, reg, probs.Unauthorized("Cannot use a non-valid registration")
```

Comment:
This is the general catch for all authenticated endpoints, so the message indicates you tried to do another action, e.g. `new-authz`, but with a deactivated (or in the future, revoked) registration.